homeassistant/components/moonraker/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""The moonraker integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .connector import APIConnector
+from .const import DATA_CONNECTOR, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[str] = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up moonraker from a config entry."""
+    if DOMAIN not in hass.data:
+        hass.data[DOMAIN] = {}
+
+    session = async_get_clientsession(hass)
+    connector = APIConnector(hass, session, entry)
+    hass.data[DOMAIN][entry.entry_id] = {DATA_CONNECTOR: connector}
```

Comment:
Additionally, if `DATA_CONNECTOR` is the only thing stored for this domain then we don't have to use that additional dictionary.

```suggestion
    hass.data[DOMAIN][entry.entry_id] = connector
```